packages/commitlint-issue-reference/package.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "@mediamonks/commitlint-issue-reference",
+  "version": "1.0.0-alpha.1",
+  "description": "Ensures your commit messages always include the same issue key as in your branch name",
+  "keywords": [
+    "commitlint",
+    "husky",
+    "git",
+    "commit",
+    "lint",
+    "issue",
+    "ticket",
+    "jira",
+    "github"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mediamonks/eslint-config.git"
+  },
+  "author": "frontend.monks",
+  "license": "MIT",
+  "exports": {
+    "import": "./dist/index.js"
+  },
+  "type": "module",
+  "files": [
+    "dist",
+    "bin",
+    "README.md",
+    "LICENSE"
+  ],
+  "bin": {
+    "commitlint-issue-reference": "./bin/commitlint-issue-reference.js"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "dev": "nodemon --ext ts --exec 'tsm' example/index.ts",
+    "dev:cli": "npm run build:ts -- --watch",
+    "test": "jest ./src/",
+    "build": "npm run clean && npm run build:ts",
+    "build:ts": "tsc -p ./tsconfig.build.json",
+    "clean": "npm run clean:test && npm run clean:npm",
+    "clean:test": "shx rm -rf coverage .nyc_output",
+    "clean:npm": "shx rm -rf dist",
+    "lint": "eslint --ext .js,.ts .",
+    "fix": "npm run lint -- --fix",
+    "format": "prettier \"**/*.{js,ts,md,json}\" --write --loglevel warn",
```

Comment:
I think it's better for this repo to have prettier configured at the root. Prettier can also format the `yml` files in the .github folder if the configuration is moved.